### PR TITLE
[Bug][Snackoo] Fix staff multi-select #7658

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/7658

UI Changes only

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]
good catch! literally catching a drop down float higher and higher into the window

**Review by frankhereford — APPROVED** [review]
Works and looks great! 🚀 🚢

**Review by patrickm02L — APPROVED** [review]
Works like a dream! Thanks for the comments.

**Review by amenity — APPROVED** [review]
So well-behaved now! Thanks, @sergiogcx

File: moped-editor/src/views/projects/projectView/ProjectTeamRoleMultiselect.js [hunk 1]
```diff
@@ -67,7 +56,18 @@ const ProjectTeamRoleMultiselect = ({
         onChange={handleChange}
         input={<Input id="select-multiple" />}
         renderValue={selected => selected.map(value => roles[value]).join(", ")}
-        MenuProps={MenuProps}
+        /*
+            There appears to be a problem with MenuProps in version 4.x (which is fixed in 5.0),
+            this is fixed by overriding the function "getContentAnchorEl".
+                Source: https://github.com/mui-org/material-ui/issues/19245#issuecomment-620488016
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectTeamRoleMultiselect.js:62]: 👍

**Review by mateoclarke — COMMENTED** [review]
🪄 ✨ 🐛 ✨ 🦋

**Pull request merged**